static/app/views/integrationOrganizationLink/index.tsx

```diff
@@ -54,6 +55,8 @@ export default class IntegrationOrganizationLink extends DeprecatedAsyncView<
   State
 > {
   disableErrorReport = false;
+  // TODO: stop using control silo which is dependent on figuring out how to
+  // check the Github installation data which is on the control silo
```

Comment:
Is this comment for the org details request that is being made? The integration configuration request will have to continue going to control silo as the regions cannot handle that request.

Reply:
@markstory when we call that endpoint from the rest of the app, we hit the region silo, no?